### ISSUE TITLE
Next message not announced after previous message delete

### ISSUE THREAD

**Issue by sahmaunt** [kellylford/QuickMail#157]
When a Inbox message is deleted, the next message is not announced. Very small issue with easy workaround of down and up arrows. Using Windows 11 and NVDA (version (rVersion	10.0.26200 Build 26200) 2026.1.1 (2026.1.1.55980))

## Steps to reproduce

1. Delete a message in the inbox
2. 
3. 

## Expected behaviour

the next message should be annouced
## Actual behaviour
silence

## Environment

- Quick0.7.6Mail version:
- Windows rVersion	10.0.26200 Build 26200version:
- Screen reader (if relevant):NVDA 2026.1.1 (2026.1.1.55980)

**Comment by sahmaunt** [comment]
Love this software. The Google account authentication is a little scary but not the end of the world. Very intuitive with just enough extra features. Thank you. I am switching from Thunderbird.

**Comment by kellylford** [comment]
@sahmaunt So far I've not been able to reproduce this but am continuing to try.

**Comment by sahmaunt** [comment]
Hi Kelly,


I can't reproduce it either. Maybe I just needed to restart NVDA. This is my first bug report on any program and I probably jumped the gun. Let's call it solved. Thank you for a great and promising product.   


Joel Sahmaunt
